modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_eu.properties

```diff
@@ -13425,7 +13425,7 @@ scheduled-events=Antolaturiko gertaerak
 scheduled-jobs=Scheduled Jobs (Automatic Copy)
 scheduled-publish-layout-local=Scheduled Local Publish (Automatic Copy)
 scheduled-publish-layout-remote=Scheduled Remote Publish (Automatic Copy)
-scheduled-start-date=Scheduled Start Date (Automatic Copy)
+scheduled-start-date=Programatutako hasierako data
```

Comment:
These Language files should not have been committed, it needs to be removed.

Reply:
Done!